compact/definitions.xml

```diff
@@ -461,18 +460,23 @@ Examples:
     <constant name="ForwardRICHRegion_tan1"        value="CentralTrackingRegionP_tan * 0.88" />
     <constant name="ForwardRICHRegion_tan2"        value="Eta3_6_tan * 0.89" />
 
-    <constant name="BackwardPIDRegion_length"     value="25.0*cm" />
-    <constant name="BackwardPIDRegion_zmin"       value="CentralTrackingRegionN_zmax" />
-    <constant name="BackwardPIDRegion_zmax"       value="BackwardPIDRegion_zmin+BackwardPIDRegion_length" />
-    <constant name="BackwardPIDRegion_rmax"       value="64.0*cm" /> <!-- FIXME hardcoded, was CentralTrackingRegion_rmax -->
+    <constant name="MRICH_length"                 value="25.0*cm" />
+    <constant name="PFRICH_length"                value="54.1*cm" />
+    <constant name="BackwardPIDRegion_length"     value="max(MRICH_length,PFRICH_length)" /> <!-- FIXME: should this depend on brycecanyon vs. arches -->
```

Comment:
This is something that should depend on `arches` vs. `brycecanyon`, I'm not sure what to do about it, so set it to `max` of the two RICH lengths for now.

Reply:
No longer relevant